tests/integrationtest/r/planner/core/integration_partition.result

```diff
@@ -739,29 +739,39 @@ a	b	c
 10	2022-01-01 00:00:00	Wowe
 explain format = 'brief' select * from t where a = 10 and b = "2022-01-01" and c = "Wow";
 id	estRows	task	access object	operator info
-  └─TableFullScan	12.00	cop[tikv]	table:t	keep order:false
-TableReader	0.52	root	partition:p3	data:Selection
-└─Selection	0.52	cop[tikv]		eq(rangecolumnsmulti.t.a, 10), eq(rangecolumnsmulti.t.b, 2022-01-01 00:00:00.000000), eq(rangecolumnsmulti.t.c, "Wow")
+      └─TableFullScan	10000.00	cop[tikv]	table:t	keep order:false, stats:pseudo
```

Comment:
The result looks strange here. We shoud remove `--sorted_result` in `integration_partition.test`

Reply:
Done